src/PanoViewer/PanoViewer.js

```diff
@@ -907,7 +907,7 @@ export default class PanoViewer extends Component {
 	 * @static
 	 */
 	static isWebGLAvailable() {
-		return WebGLUtils.isWebGLAvailable();
+		return WebGLUtils.isWebGLAvailable() && WebGLUtils.isStableWebGL();
```

Comment:
@jongmoon  Is isStableWebGL just webgl, not experiment-webgl?

Reply:
- `isWebGLAvailable`: just webgl
  - `isStableWebGL`: check if above android 4.4 chrome

Reply:
@jongmoon Is isWebGLAvailable in the constructor OK?

Reply:
@jongmoon Of course~

Reply:
@jongmoon LGTM~!

Reply:
@younkue There would be change for this function. This function will be restored and `isSupported` function will be added to provide `WebGLUtils.isWebGLAvailable() && WebGLUtils.isStableWebGL();`

Reply:
@jongmoon  ```isSupported``` is much more intuitive and popular.

Reply:
@jongmoon  Do you use isWebGLAvailable?